boards/atmega1284p/include/board.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2014 Freie Universität Berlin, Hinnerk van Bruinehsen
+ *               2016 Laurent Navet <laurent.navet@gmail.com>
+ *               2019 Otto-von-Guericke-Universität Magdeburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_atmega1284p
+ * @{
+ *
+ * @file
+ * @brief       Board specific definitions for the standalone ATmega1284P "board"
+ *
+ * @author      Marian Buschsieweke <marian.buschsieweke@ovgu.de>
+ * @author      Hinnerk van Bruinehsen <h.v.bruinehsen@fu-berlin.de>
+ * @author      Laurent Navet <laurent.navet@gmail.com>
```

Comment:
I think you can drop all but your name here.
IMHO this is more about who is responsible for the board than to properly attribute the boilerplate :wink:

Reply:
This is actually the same I interpret this. But I had some discussions about this; and the take away to me was to better just keep any author tags when you basically copy files, as some people seem to be sensitive about their name being removed.